target/linux/generic/files/drivers/net/phy/ar8327.c

```diff
@@ -926,10 +926,21 @@ ar8327_setup_port(struct ar8xxx_priv *priv, int port, u32 members)
 
 	t = pvid << AR8327_PORT_VLAN0_DEF_SVID_S;
 	t |= pvid << AR8327_PORT_VLAN0_DEF_CVID_S;
+
+	if (priv->vlan && priv->port_vlan_prio[port] > 0) {
+		u32 prio = priv->port_vlan_prio[port] & AR8327_PORT_VLAN0_DEF_PRI_MASK;
+		t |= prio << AR8327_PORT_VLAN0_DEF_SPRI_S;
```

Comment:
add a blank line here and move the "& AR8327_PORT_VLAN0_DEF_PRI_MASK" into the set function please

Reply:
done